src/html/Options.html

```diff
@@ -70,7 +70,7 @@ <h5 data-i18n="newtab_revoked"></h5>
                        
 
                    
-                    <div id="admDivCheck" style="display:none;"></div>
+                    <div id="admDivCheck" style="display:none;">

```

Comment:
`<div ......>` is the beginning of the div element, and `</div>` is the ending. This div needs to contains the 6 elements below, so the `</div>` must be moved after them.

Reply:
Yes, also fixed after pushed my last changes on the repo.